content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,101 @@
-publishdate: 2019-03-01
+publishdate: 2019-03-12
 icon: notes
 ---
 
-TODO announcement
+We're proud to release Istio 1.1! It's been a while since 1.0 and we've done some significant improvements to make
+the overall product more reliable, more efficient, and more scalable.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.1.
 
 {{< relnote_links >}}
 
-## Policies and Telemetry
+## Upgrades
+
+- **Helm Changes**. 
+TBD: need full content & links
+
+- **TBD: TITLE**, Control plane and data plane requested resource changes
+TBD: need full content & links
+
+## Traffic management
+
+- **Allow All Outbound Traffic**. Allow all outbound traffic by default now.   To enforce no outbound traffic at mesh wide, configure ```global.outboundTrafficPolicy.mode``` to ```REGISTRY_ONLY```
+
+- **Egress Gateway Disabled By Default**.- Egress gateway is disabled by default now.  To enable it, configure gateways.istio-egressgateway.enabled to ```true```
 
-- **Kiali**. The Service Graph addon has been [deprecated](https://github.com/istio/istio/issues/9066) in favor of [Kiali](https://www.kiali.io). See the [Kiali Task](/docs/tasks/telemetry/kiali/) for more details about Kiali.
+- **Simpler Service Entry Resources**. Service entry resources are greatly simplified by reducing the need to bind virtual services to them.
+
+- TBD: note about sidecar and export_to API.
 
 ## Security
 
-- Deprecated `RbacConfig` replacing it with `ClusterRbacConfig` to implement the correct cluster scope.
-  Refer to our guide on [Migrating the `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/#migrating-the-rbacconfig-to-clusterrbacconfig)
-  for migration instructions.
+- **Readines and Liveness Probes**. Istio now supports Kubernetes' HTTP readiness and liveness probes when mutual TLS is enabled.
+TBD: LINK TO DOC
+
+- **`ClusterRbacConfig`**. The `RbacConfig` resource is being replaced with the `ClusterRbacConfig` resource to implement the correct cluster scope.
+Refer to our guide on [Migrating the `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrading-istio#migrating-the-rbacconfig-to-clusterrbacconfig)
+for migration instructions.
+
+## Multiclusters
+
+- **Non-Routable L3 Networks**. A single Istio control plane can now be used in multicluster environments with non-routable
+L3 networks.
+TBD: LINK
+
+- **Multiple Control Planes**. Multiple Istio control plane multicluster is now supported.
+TBD: LINK
+
+## Policies and telemetry
+
+- **Policy Checks Off By Default**. Policy checks are now turned off by default which improves performance for most customer scenarios.
+TBD: LINK
+
+- **Kiali**. [Kiali](https://www.kiali.io) replaces the [Service Graph addon](https://github.com/istio/istio/issues/9066) and provides
+a richer visualization experience. See the [Kiali Task](/docs/tasks/telemetry/kiali/) for more details.
+
+- **Reduced Overhead**. There have been several performance and scale improvements including:
```

Comment:
worth calling out other perf improvements?  (work with some of our default adapters, including fluentd, work with our dependencies including trace libraries, prometheus, etc.)

Reply:
Yes, definitely worth it. Could you or @mandarjog  suggest the right text please?